### PR TITLE
Removing Mobile Services from Current Versions

### PR DESCRIPTION
**Pull request by calebk1**
This PR remove Mobile Services from this repository, as it's now deprecated.

### CODE REVIEW

File: src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md [hunk 1]
```diff
@@ -73,8 +73,6 @@ If SDK privacy status is `optedout`, the push identifier will not be set.
 
 2. In `application:didRegisterForRemoteNotificationsWithDeviceToken:`, call `setPushIdentifier` to set the Push identifier.
 
-   For more information, see [setPushIdentifier](../../adobe-analytics-mobile-services/index.md#set-up-push-messaging).
```
Comment (emdobrin) [inline src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md:76]: this is an API in MobileCore, let's redirect to that instead

File: src/pages/documentation/v4-end-of-life-faq.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ To upgrade to the Experience Platform SDKs, see [Upgrade to the Experience Platf
 
 ### Will Mobile Services functionality still work?
 
-Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the [Adobe Analytics - Mobile Services](./adobe-analytics-mobile-services/index.md) extension for the new Experience Platform Mobile SDKs.
+Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the [Adobe Analytics - Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension for the new Experience Platform Mobile SDKs.
```
Comment (emdobrin) [inline src/pages/documentation/v4-end-of-life-faq.md:42]: can we reword this to indicate this extension is supported in previous versions of the SDK?

File: src/pages/documentation/adobe-analytics-mobile-services/release-notes.md [hunk 3]
```diff
@@ -2,26 +2,8 @@
 
 Release notes and change logs for the Adobe Mobile Services extension
```
Comment (emdobrin) [inline src/pages/documentation/adobe-analytics-mobile-services/release-notes.md:3]: are we not removing this entire folder?

File: src/pages/documentation/v4-end-of-life-faq.md (hunk 2 above)
Comment (calebk1) [reply src/pages/documentation/v4-end-of-life-faq.md:42]: That makes sense!

File: src/pages/documentation/adobe-analytics-mobile-services/release-notes.md [hunk 4]
```diff
@@ -1,27 +0,0 @@
-# Release Notes
-
-Release notes and change logs for the Adobe Mobile Services extension
-
-## November 5, 2021
-
-### iOS AEPMobileServices 3.0.3
```
Comment (emdobrin) [inline src/pages/documentation/adobe-analytics-mobile-services/release-notes.md:7]: can we preserve these in previous versions?

Comment (calebk1) [reply src/pages/documentation/adobe-analytics-mobile-services/release-notes.md:7]: IIRC - I kept them in the previous versions, but I'll double check and let you know if I did not.

Comment (calebk1) [reply src/pages/documentation/adobe-analytics-mobile-services/release-notes.md:7]: I did not! I'll go re-add them.

Comment (calebk1) [reply src/pages/documentation/adobe-analytics-mobile-services/release-notes.md:7]: Or, sorry - I preserved it within the Mobile Services folder, but not in the release-notes folder. So I'll add those there.

File: src/pages/documentation/current-sdk-versions.md [hunk 5]
```diff
@@ -30,6 +30,12 @@ Due to sunset of [JCenter by JFrog](https://jfrog.com/blog/into-the-sunset-bintr
...
 | [Identity for Edge Network](../identity-for-edge-network/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeidentity.svg?logo=android&logoColor=white&label=edgeidentity&style=flat-square)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeidentity) | [Link](https://github.com/adobe/aepsdk-edgeidentity-android) |
 | [Consent for Edge Network](../consent-for-edge-network/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeconsent.svg?logo=android&logoColor=white&label=edgeconsent&style=flat-square)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeconsent) | [Link](https://github.com/adobe/aepsdk-edgeconsent-android) |
+| [Adobe Experience Platform Assurance](../platform-assurance-sdk/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/assurance.svg?logo=android&logoColor=white&label=assurance&style=flat-square)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/assurance) | [Link](https://github.com/adobe/aepsdk-assurance-android) |
```
Comment (emdobrin) [inline src/pages/documentation/current-sdk-versions.md:33]: the merge on this file does not look right, can you revert the changes on this file?

File: src/pages/documentation/upgrade-platform-sdks/comparison.md [hunk 6]
```diff
@@ -38,9 +38,6 @@ The following tables provide information about the differences between the new E
 | Postbacks - Get/POST URL requests | Yes |	Yes - [Signals](../mobile-core/signal/index.md) extension |
 | Postbacks - PII Get/POST URL requests | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
 | Postbacks - Open app deeplink | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
-| Push Messaging | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension | 
-| In-app Messaging | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension |
-| Marketing/Acquisition Links | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension |
```
Comment (emdobrin) [inline src/pages/documentation/upgrade-platform-sdks/comparison.md:43]: can we keep these rows for now and link to previous versions?

File: src/pages/documentation/v4-end-of-life-faq.md [hunk 7]
```diff
@@ -39,7 +39,7 @@ To upgrade to the Experience Platform SDKs, see [Upgrade to the Experience Platf
 
 ### Will Mobile Services functionality still work?
 
-Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the [Adobe Analytics - Mobile Services](./adobe-analytics-mobile-services/index.md) extension for the new Experience Platform Mobile SDKs.
+Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the Adobe Analytics - Mobile Services extension for the Experience Platform Mobile SDKs. Documentation for this can be found in the [previous versions of our documentation](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/).
```
Comment (emdobrin) [inline src/pages/documentation/v4-end-of-life-faq.md:42]: previous versions of the Mobile SDKs

**Review by emdobrin — APPROVED** [review]

File: src/pages/documentation/upgrade-platform-sdks/comparison.md [hunk 8]
```diff
@@ -38,9 +38,9 @@ The following tables provide information about the differences between the new E
...
+| Push Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
+| In-app Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
+| Marketing/Acquisition Links | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
```
Comment (swarna04) [inline src/pages/documentation/upgrade-platform-sdks/comparison.md:43]: Shouldn't these link to AA - Mobile Services in previous versions instead
https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/

File: src/pages/documentation/v4-end-of-life-faq.md [hunk 9]
```diff
@@ -39,7 +39,7 @@ To upgrade to the Experience Platform SDKs, see [Upgrade to the Experience Platf
 
 ### Will Mobile Services functionality still work?
 
-Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the [Adobe Analytics - Mobile Services](./adobe-analytics-mobile-services/index.md) extension for the new Experience Platform Mobile SDKs.
+Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging with version 4 SDKs or by leveraging the Adobe Analytics - Mobile Services extension for the Experience Platform Mobile SDKs. Documentation for this can be found in the [previous versions of the Mobile SDKs](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/).
```
Comment (swarna04) [inline src/pages/documentation/v4-end-of-life-faq.md:42]: nit: Suggest minor rephrase - The documentation for this extension is available in the [previous versions of the Mobile SDKs](...).

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md [hunk 10]
```diff
@@ -61,7 +61,7 @@ If SDK privacy status is `optedout`, the push identifier will not be set.
 
    ![ECID network request for push token sync](../assets/push-notifications/push-identifier.png)
 
-<Variant platform="ios" task="receive" repeat="5"/>
+<Variant platform="ios" task="receive" repeat="4"/>
```
Comment (swarna04) [inline src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md:64]: Missed this before! 
1. Preserve the indentation on L76 and you do not need to modify the repeat count. (Recommend this as this is in-line with formatting on the rest of the page)
2. If you remove the indentation then repeat count will have to be increased to 6 (and not decreased to 4). Currently, there is an issue with the content in iOS/ Android tabs because of removal of indentation and decrease of repeat count.

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md (hunk 10 above)
Comment (emdobrin) [reply src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md:64]: @calebk1 can you take this one?

Comment (calebk1) [reply src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md:64]: I think we I tested the repeat before without the removal of indents, it wasn't working - which is why I removed it in this PR (because there was SOMETHING weird going on).

Comment (calebk1) [reply src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md:64]: OK, it's all working now so ???

**Review by emdobrin — APPROVED** [review]

**Review by swarna04 — APPROVED** [review]
